src/api/products.ts

```diff
@@ -6,3 +6,10 @@ export const getProducts = async () => {
 
   return products || null;
 };
+
+export const getProductById = async (productId: number) => {
+  const products = await getProducts();
```

Comment:
please use separate route for getting product data provided by Api https://api.thedogapi.com/v1/breeds/2 instead of mapping in by yourself from the full collection

Reply:
@IgorNovikov10 ok, but this URL (https://api.thedogapi.com/v1/breeds/2) gives all info about the product except the image URL. So in this case we need to make another request for an image. Is it a better solution than now?

Reply:
Yes, make another request to get the image

In the current solution you are performing another extra request to get the whole list of products. It's not needed on the product page